lib/Service/SynchronizationService.php

```diff
@@ -47,6 +47,9 @@ class SynchronizationService
     private ObjectService $objectService;
     private Source $source;
 
+    const SINGLE_ENDPOINT            = 'singleEndpoint';
+    const MERGE_DATA_SINGLE_ENDPOINT = 'mergeDataSingleEndpoint';
```

Comment:
I was advised from AI to make these fixed keys const's for better code maintanability and less risk of typos